serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx

```diff
@@ -42,3 +42,34 @@ After receiving no query for 5 minutes, a Serverless SQL Database will enter int
 
```

Comment:
I find it a bit weird to have the specification table as a first content. I would put it at the end of the Autoscaling part.